code/modules/power/lighting.dm

```diff
@@ -203,13 +201,13 @@
 
 /obj/machinery/light/LateInitialize()
 	var/area/A = get_area(src)
-	seton(A.lightswitch && A.power_light)
+	turn_light(null, (A.lightswitch && A.power_light))
 
 
 /obj/machinery/light/Destroy()
 	var/area/A = get_area(src)
 	if(A)
-		on = FALSE
+		light_on = FALSE
 	return ..()
```

Comment:
why does this exist?
im pretty sure this does nothing

Reply:
i'm also pretty sure this does nothing, but i can't assure it. It was there before, I wasn't bold enough to remove it